pages/watches-support.json

```diff
@@ -400,6 +419,7 @@
         "Sony Smartwatch 3"
       ],
       "status":"experimental",
+      "maintainers": [ ],
```

Comment:
@FlorentRevest was the original maintainer.
I think marking this watch as *unmaintained* again is not right since it had recent work that improved support (battery life fix).